startables/startables.py

```diff
@@ -73,16 +73,25 @@ def get_local_file(self) -> pathlib.Path:
 
 
 class ColumnMetadata:
-    def __init__(self, unit: Unit, home_unit: Optional[Unit] = None, remark: Optional[str] = None):
+    def __init__(self, unit: Unit, home_unit: Optional[Unit] = None, remark: Optional[str] = None, format_str=None):
+        '''
+
+        :param unit:
```

Comment:
I don't really know how to describe these parameters because I don't 100% understand them!

Reply:
Fair enough. I should write docstrings for those parameters--and for the whole class